src/tests/ftest/soak/soak_utils.py

```diff
@@ -571,7 +582,11 @@ def launch_server_stop_start(self, pools, name, results, args):
                 self.log.error(
                     "<<<FAILED:dmg system start failed", exc_info=error)
                 status = False
+            for pool in pools:
+                self.dmg_command.pool_query(pool.uuid)
             if status:
+                # Wait ~ 30 sec before issuing the reintegrate
```

Comment:
Are we waiting 30 sec or 180 sec?

Reply:
30 sec; I will change that 180 was a debug change